### PR TITLE
add junit test for RpcConfigChangeNotifierTest

### PR DESCRIPTION
**Pull request by Chaos-coder**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

for  #5092

## Brief changelog

add junit test for RpcConfigChangeNotifierTest

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    
+    @Test
+    public void testOnEvent() {
+        final String groupKey = GroupKey2.getKey("nacos.internal.tps.control_rule_1", "nacos", "tenant");
+        final String limitGroupKey = GroupKey2.getKey("nacos.internal.tps.nacos.internal.connection.limit.rule",
+                "nacos", "tenant");
+        List<String> betaIps = new ArrayList<>();
+        betaIps.add("1.1.1.1");
+        rpcConfigChangeNotifier.onEvent(new LocalDataChangeEvent(groupKey, true, betaIps));
+        rpcConfigChangeNotifier.onEvent(new LocalDataChangeEvent(limitGroupKey));
```
Comment (li-xiao-shuang) [inline config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java:59]: 应该验证下结果

File: config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.alibaba.nacos.config.server.remote;
```
Comment (li-xiao-shuang) [inline config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java:1]: 加下开源协议

Comment (Chaos-coder) [reply config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java:1]: 好的

File: config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java (hunk 1 above)
Comment (Chaos-coder) [reply config/src/test/java/com/alibaba/nacos/config/server/remote/RpcConfigChangeNotifierTest.java:53]: 返回值是void，InternalConfigChangeNotifierTest类好像也无法验证结果，用户只能观察结果输出？

**Pull request closed without merging**